WordPress/UITests/Tests/PostTests.swift

```diff
@@ -33,8 +33,13 @@ class PostTests: XCTestCase {
             .closePostSettings()
             .post(action: .schedule)
 
-        try MySiteScreen()
-            .goToMoreMenu()
+        // On iPad, the menu items are already listed on screen, so we don't need to tap More Menu button
+        if XCUIDevice.isPhone {
+            try MySiteScreen()
+                .goToMoreMenu()
+        }
```

Comment:
Oh yes, completely agree. I was flip-flopping between the two and found that when this was on the screen level, there was still some flakiness (because every time a new screen is called, it's a bit of a "2-step" - the screen `init` will be called first then the actual method the test is looking for). 

That said, I think I'll give this another try because previously, during screen `init`, we were using an element that's pretty deep in the element tree (which we have since learned [can cause flakiness in the form of timeouts](https://github.com/woocommerce/woocommerce-ios/pull/10752)) with this new knowledge I think we can move this and make it work.

Reply:
Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/21627/commits/b9e05b07df0150d34f3fb9333c3f09b4a209ff30 and waiting for CI to complete the test 🤞